### ISSUE TITLE
object coerce is not exported

### ISSUE THREAD

**Issue by dpryan79** [acidgenomics/basejump#204]
I'm not sure if this is an actual issue with pipette or whether the wrong `coerce()` is just being used, but either way the following is the error that occurs when trying to install the most recent release.

```
10:57:51 BIOCONDA INFO (OUT) Error: object 'coerce' is not exported by 'namespace:pipette'
10:57:51 BIOCONDA INFO (OUT) Execution halted
10:57:51 BIOCONDA INFO (OUT) ERROR: lazy loading failed for package ‘basejump’
10:57:51 BIOCONDA INFO (OUT) * removing ‘/opt/conda/conda-bld/r-basejump_1579863178045/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/R/library/basejump’
```

**Comment by mjsteinbaugh** [comment]
Thanks I’ll take a look!

**Comment by dpryan79** [comment]
Xref: https://github.com/bioconda/bioconda-recipes/pull/19943

**Comment by mjsteinbaugh** [comment]
This should be fixed in pipette v0.4.1. Thanks @dpryan79

**Comment by mjsteinbaugh** [comment]
Argh still broken even with the pipette update. I'll patch basejump today.
